Calling/ClientApp/src/App.tsx

```diff
@@ -78,8 +79,7 @@ const App = () => {
     } else if (page === 'endCall') {
       return (
         <EndCall
-          message={ store.getState().calls.attempts > 3 ? 'Unable to join the call' : 
-          'You left the call'}
+          message={'You left the call'}
```

Comment:
Nit: If the other person left the call, will he get the same message?

Reply:
not everyone gets thrown out of a call when someone leaves so only the people that will get this message is the one that hits the leave button.